test/cholmod.jl

```diff
@@ -1,8 +1,12 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
 module CHOLMODTests
-
 using Test
+
+@static if !Base.USE_GPL_LIBS
+    @info "Not use GPL libs, Skipping CHOLMOD Tests"
```

Comment:
Could you update the message to `"This Julia build excludes the use of SuiteSparse GPL libraries. Skipping CHOLMOD tests"`?